torchbenchmark/models/nvidia_deeprecommender/__init__.py

```diff
@@ -0,0 +1,93 @@
+import torch
+import torch.optim as optim
+import torchvision.models as models
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import RECOMMENDATION
+
+import gc
+from .nvtrain import DeepRecommenderTrainBenchmark
+from .nvinfer import DeepRecommenderInferenceBenchmark
+
+class Model(BenchmarkModel):
+  
+  task = RECOMMENDATION.RECOMMENDATION
+  
+  def __init__(self, device="cpu", jit=False):
+    super().__init__()
+    self.devicename = device
+    self.notimplementedreason = "Implemented"
+    self.evalMode = True #default to inference
+
+    if jit:
+      self.notimplementedreason = "Jit Not Supported"
+
+    elif self.devicename != "cpu" and self.devicename != "cuda":
+      self.notimplementedreason = "device type not supported"
+
+    elif self.devicename == "cuda" and torch.cuda.is_available() == False:
+      self.notimplementedreason = "cuda not available on this device"
+
+    else:
+      self.train_obj = DeepRecommenderTrainBenchmark(device = device, jit = jit)
```

Comment:
Could you please unify the style of naming of instance variables? The style of pytorch is to use underscore style.

Reply:
done